lib/errors/index.ts

```diff
@@ -34,3 +35,12 @@ export interface NoRefImageError {
     currImg: ImageFile;
     refImg: ImageFile;
 }
+
+export interface InvalidRefImageError {
+    name: ErrorNames['INVALID_REF_IMAGE'];
+    stateName: string;
+    message: string;
+    stack?: string;
+    currImg: ImageFile;
+    refImg: ImageFile;
```

Comment:
Why is it required, just like in "NoRefImageError"?
Feels like this property shouldn't exist.
Not important, though

Reply:
Because errors thrown by Testplane always have this field. In this case refImg stores path to ref image that turned out to be invalid. IMO a useful piece of info.